web-wallet/src/lib/dusk/components/Tabs/Tabs.css

```diff
@@ -18,8 +18,15 @@
   user-select: none;
   cursor: default;
   white-space: nowrap;
-  overflow: hidden;
+  overflow-x: scroll;
   width: 100%;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+  overscroll-behavior: contain;
```

Comment:
I don't have a strong preference on this one – we can have it or remove it.

Personally, I think not having it is better for accessibility, but having it can also improve the experience slightly for the rest of the users.

On the other hand, seems like Chrome and Safari have implemented this differently and I think we should strive to have a consistent behavior across browsers.

All in all, it doesn't really matter at the moment, as we mostly display 2 tabs at a time, so users won't have to scroll anyways, regardless of the device size. Example:

https://github.com/dusk-network/rusk/assets/34094428/d20b3589-543c-4dfc-8f8f-df6b2f511653

Reply:
Yeah, that's true too. I'm happy with regardless, whatever you, guys, decide

Reply:
It seems that the [`scrollbar-width`](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width) is a bit too recent to be used here.

Personally I would go back to `overflow: hidden` for the time being.

cc @kieranhall @deuch13

Reply:
> It seems that the [`scrollbar-width`](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width) is a bit too recent to be used here.
> 
> Personally I would go back to `overflow: hidden` for the time being.
> 
> cc @kieranhall @deuch13

Not really, as the `::-webkit-scrollbar` has been supported in ages. 

<img width="810" alt="Screenshot 2024-03-12 at 12 40 15" src="https://github.com/dusk-network/rusk/assets/34094428/7d2e3985-acba-450e-8d2e-baf947b9ff23">

As stated on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar):

<img width="808" alt="Screenshot 2024-03-12 at 12 41 46" src="https://github.com/dusk-network/rusk/assets/34094428/a5f02487-a207-43ce-98d1-47d33bd62f35">

So the combination of the properties `display: none` for the pseudo-selector -webkit-scrollbar and the `scrollbar-width: none` should cover all browsers.

Reply:
Ok, don't like proprietary things, but I'm fine with `::-webkit-scrollbar`.

Let's hear from the others.

Reply:
It will be safe to remove the `::-webkit-scrollbar` a few months after Safari rolls out support of it.

It seems like the [PR](https://github.com/WebKit/WebKit/pull/13992) has already been merged into WebKit's repo and the support is currently under a feature flag in Safari.

<img width="1017" alt="Screenshot 2024-03-12 at 12 58 35" src="https://github.com/dusk-network/rusk/assets/34094428/dcf37e70-9d0c-42ee-8ac5-bd779fe38641">

Reply:
Have you checked if this is taken care of in https://github.com/postcss/autoprefixer ?  We are using this.

Reply:
There is almost nothing for autoprefixer to do here.
Only thing it does is adding `-ms-scroll-chaining: none;` when you use `overscroll-behavior: contain;`.

Reply:
> It seems that the [`scrollbar-width`](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width) is a bit too recent to be used here.
> 
> Personally I would go back to `overflow: hidden` for the time being.
> 
> cc @kieranhall @deuch13

As long as we also have this in our css I think we are good 

```
.dusk-tabs-list::-webkit-scrollbar {
  display: none;
}
```